draft-irtf-cfrg-frost.md

```diff
@@ -821,10 +820,9 @@ to the description in {{dep-pog}}. Validation steps for these functions
 are described for each the ciphersuites below. Future ciphersuites MUST
 describe how input validation is done for DeserializeElement and DeserializeScalar.
 
-We include explicit `verify()` functions for each ciphersuite that should also
-apply to any singleton signature as well as signatures generated by FROST. By
-definition, this function assumes that all group Elements passed as input,
-including the signature R component and public key, belong to prime-order group G.
+Each ciphersuite includes explicit instructions for verifying signatures produced
+by FROST. Note that these instructions are equivalent to those produced by a single
+signer.
```

Comment:
I have a suggested rewording on the base PR related to this, yours might be better language // @chelseakomlo 

https://github.com/cfrg/draft-irtf-cfrg-frost/pull/219/files#r918194305

<img width="622" alt="image" src="https://user-images.githubusercontent.com/552961/178333603-b107fe5a-f3a0-453d-90eb-6368c54244d5.png">